bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceConsoleCommandExtension.java

```diff
@@ -44,7 +44,8 @@ public VoiceConsoleCommandExtension() {
 
     @Override
     public List<String> getUsages() {
-        return Arrays.asList(new String[] { buildCommandUsage(SUBCMD_SAY + " <text>", "speaks a text"),
+        return Arrays.asList(new String[] {
+                buildCommandUsage(SUBCMD_SAY + " <sink> <text>", "speaks a text through the given audio sink"),
```

Comment:
mmhm.. I tend not follow that as I have a scenario in mind whereby I want to redirect say output to different sinks (read: sonos zone players in different rooms). Maybe it is through for the console, but definitely not for ActionService.... Let's see.